uploadNeuroDB/minc_deletion.pl

```diff
@@ -0,0 +1,282 @@
+#! /usr/bin/perl
+use strict;
+use warnings;
+use Carp;
+use Getopt::Tabular;
+use FileHandle;
+use File::Basename;
+use File::Temp qw/ tempdir /;
+use File::Path qw/ make_path /;
+use Data::Dumper;
+use FindBin;
+use Cwd qw/ abs_path /;
+
+# These are the NeuroDB modules to be used
+use lib "$FindBin::Bin";
+use NeuroDB::File;
+use NeuroDB::MRI;
+use NeuroDB::DBI;
+use NeuroDB::Notify;
+use NeuroDB::MRIProcessingUtility;
+
+my $sessionfilesfound = '';
+my $versionInfo = sprintf "%d revision %2d", q$Revision: 1.2 $
+    =~ /: (\d+)\.(\d+)/;
+my $profile   = '';      # this should never be set unless you are in a
+                             # stable production environment
+my $seriesuid = '';
+my $fileid    = '';
+my $sth;
+my $rvl;
+my $query     = '';
+my $selORdel  = '';
+my @opt_table = (
+                 ["Basic options","section"],
+                 ["-profile     ","string",1, \$profile,
+                  "config file in ../dicom-archive/.loris_mri"
+                 ],
+                 ["-seriesuid", "string", 1, \$seriesuid, "Only deletes this SeriesUID"
+                 ],
+                 ["-fileid", "string", 1, \$fileid, "Only deletes this FileID"
+                 ],
+                 );
+
+
+my $Help = <<HELP;
+*******************************************************************************
+Minc Deletion
+*******************************************************************************
+
+Author  :   Gregory Luneau
+Date    :   July 2016
+Version :   $versionInfo
+
+
+The program does the following:
+
+Deletes minc files from the web interface by:
+  - Moving the existing files to an archive directory.
+    .mnc .jpg .header .raw_byte.gz
+  - Deleting all related data from 4 database tables.
+    parameter_file, files_qcstatus, feedback_mri_comments, files
+
+HELP
+my $Usage = <<USAGE;
+usage: $0 [options] select|confirm
+       $0 -help to list options
+
+USAGE
```

Comment:
@gluneau 
nice help section!
It would be nice to explain what the select and confirm options are supposed to do. It is not very clear.